sql-plugin/src/main/scala/org/apache/spark/sql/rapids/AggregateFunctions.scala

```diff
@@ -388,9 +388,9 @@ class CudfMergeLists(override val dataType: DataType) extends CudfAggregate {
 }
 
 /**
- * Spark handles NaN's equality by different way for non-nested float/double and float/double 
- * in nested types. When we use non-nested versions of floats and doubles, NaN values are 
- * considered unequal, but when we collect sets of nested versions, NaNs are considered equal 
+ * Spark handles NaN's equality by different way for non-nested float/double and float/double
+ * in nested types. When we use non-nested versions of floats and doubles, NaN values are
+ * considered unequal, but when we collect sets of nested versions, NaNs are considered equal
```

Comment:
```suggestion
  // Spark handles NaN's equality differently when dealing with nested and non-nested float/double.
  // When we use non-nested versions of floats and doubles, NaN values are
  // considered unequal, but when we collect sets of nested versions, NaNs are considered equal
```

Reply:
This is nothing related to the work in this PR. Just the spaces in each line are removed automatically by my IDE :D